browser/ui/tabs/test/BUILD.gn

```diff
@@ -19,8 +19,52 @@ source_set("browser_tests") {
     "//chrome/browser",
     "//chrome/browser/profiles:profile",
     "//chrome/browser/ui",
-    "//chrome/test:test_support_ui",
     "//components/prefs",
     "//content/test:test_support",
   ]
+
+  if (!is_android) {
+    deps += [ "//chrome/test:test_support_ui" ]
+  }
+}
+
+source_set("unit_tests") {
+  testonly = true
+
+  defines = [ "HAS_OUT_OF_PROC_TEST_RUNNER" ]
+
+  if (!is_android) {
+    sources = [
+      "split_view_browser_data_unittest.cc",
+      "split_view_tab_strip_model_adapter_unittest.cc",
+    ]
+
+    deps = [
+      ":test_support",
+      "//brave/browser/ui/tabs",
+      "//chrome/browser",
+      "//chrome/browser/ui",
+      "//chrome/test:test_support",
+      "//chrome/test:test_support_ui",
+      "//components/tab_groups",
+    ]
+  }
+}
+
+# In order to resolve circular deps to the test_tab_strip_model_delegate,
+# declares source set here instead of adding these files to //brave/test/BUILD.gn
+source_set("test_support") {
+  if (!is_android) {
```

Comment:
Do not use `!is_android` like this. If this is toolkit_views it should be under the toolkit_views views buildflag and should be in browser/ui/views

Reply:
The upstream test target also includes this in `!is_android` https://source.chromium.org/chromium/chromium/src/+/main:chrome/test/BUILD.gn;l=7954;drc=1af2f445d25a6f2e12984418df7b9a0bd09d4523` . Could you tell me why we shouldn't use `!is_android` ?

Reply:
nvm, it is under `!is_android`, but don't create an empty test support target for non-android, that is confusing. The correct way to do this is to wrap this entire target in `!is_android` so it doesn't exist at all for android